2024-09-12 18:25:12,112 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-12 18:41:48,034 - app - INFO - Cargando datos del usuario: Ricardo [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:156]
2024-09-12 18:41:48,034 - app - INFO - Mostrando imagen de perfil para el usuario Ricardo: dc8a083d08019809.jpg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:166]
2024-09-13 09:05:18,546 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:05:18,626 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:05:18,664 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:08:27,726 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:08:27,806 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:08:27,842 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:10:19,746 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:10:19,825 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:10:19,870 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:13:42,295 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:13:42,375 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:13:42,410 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:15:07,646 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:15:07,726 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:15:07,764 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:21:30,522 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:21:30,600 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:21:30,637 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:31:10,938 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:31:11,013 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:31:11,049 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:34:33,992 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:34:34,073 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:34:34,118 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:36:06,047 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:36:06,125 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:36:06,164 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:37:57,302 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:37:57,381 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:37:57,417 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:39:31,907 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:39:31,985 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:39:32,025 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:50:34,800 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:50:34,879 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:50:34,917 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 09:53:31,157 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 09:53:31,236 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 09:53:31,272 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 10:20:15,292 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/edf_catalogacion_qr/app/__init__.py:70]
2024-09-13 10:20:15,375 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:26]
2024-09-13 10:20:15,417 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/edf_catalogacion_qr/app/models.py:33]
2024-09-13 10:20:31,730 - app.utils - INFO - Correo de restablecimiento de contraseña enviado a edfrutos@gmail.com [in /home/ec2-user/edf_catalogacion_qr/app/utils.py:154]
2024-09-13 11:08:23,395 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:153]
2024-09-13 11:08:23,395 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: 0446ad9b9b79ad76.jpg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:163]
2024-09-13 11:09:00,020 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:153]
2024-09-13 11:09:00,020 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: 0446ad9b9b79ad76.jpg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:163]
2024-09-13 11:09:05,744 - app - INFO - Iniciando actualización de cuenta para el usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:110]
2024-09-13 11:09:05,745 - app - INFO - Información recibida - Username: edefrutos, Email: edfrutos@gmail.com, Address: Calle Mango, 3, Phone: 983246972 [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:111]
2024-09-13 11:09:05,787 - app - INFO - Cuenta actualizada exitosamente para el usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:138]
2024-09-13 11:09:05,858 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:153]
2024-09-13 11:09:05,859 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: 0446ad9b9b79ad76.jpg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:163]
2024-09-13 11:09:13,926 - app - INFO - Iniciando actualización de cuenta para el usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:110]
2024-09-13 11:09:13,926 - app - INFO - Información recibida - Username: edefrutos, Email: edfrutos@gmail.com, Address: Calle Mango, 3, Phone: 983246972 [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:111]
2024-09-13 11:09:13,926 - app - INFO - Actualizando foto de perfil para el usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:116]
2024-09-13 11:09:13,958 - app.utils - INFO - Foto de perfil guardada en: /home/ec2-user/edf_catalogacion_qr/app/static/profile_pics/f0de395c22d4046d.jpeg [in /home/ec2-user/edf_catalogacion_qr/app/utils.py:68]
2024-09-13 11:09:13,958 - app - INFO - Foto de perfil actualizada: f0de395c22d4046d.jpeg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:120]
2024-09-13 11:09:14,022 - app - INFO - Cuenta actualizada exitosamente para el usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:138]
2024-09-13 11:09:14,093 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:153]
2024-09-13 11:09:14,093 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: f0de395c22d4046d.jpeg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:163]
2024-09-13 15:25:38,248 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:153]
2024-09-13 15:25:38,248 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: f0de395c22d4046d.jpeg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:163]
2024-09-13 18:53:03,149 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:153]
2024-09-13 18:53:03,149 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: f0de395c22d4046d.jpeg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:163]
2024-09-13 19:17:56,888 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:153]
2024-09-13 19:17:56,889 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: f0de395c22d4046d.jpeg [in /home/ec2-user/edf_catalogacion_qr/app/routes.py:163]
